### PR TITLE
Add blog post for best presentation award

### PR DESCRIPTION
**Pull request by blegat**
We could add it to the same page as the workshop but I though a different blog post would be better. We should add the pictures before merging.

### CODE REVIEW

File: _posts/2024-07-20-prize_2024.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: page
```
Comment (odow) [inline _posts/2024-07-20-prize_2024.md:2]:
```suggestion
layout: post
```

File: _posts/2024-07-20-prize_2024.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+title:  "JuMP-dev prize 2024"
+date:   2024-07-20
```
Comment (odow) [inline _posts/2024-07-20-prize_2024.md:4]: categories: [jump-dev-prize]
author: "Benoit Legat and Joaquim Garcia"

File: _posts/2024-07-20-prize_2024.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+---
+
+The JuMP-dev 2024 workshop introduced a new best presentation prize.
```
Comment (odow) [inline _posts/2024-07-20-prize_2024.md:7]:
```suggestion
The [JuMP-dev 2024 workshop](/meetings/jumpdev2024) introduced a new best presentation prize.
```

File: _posts/2024-07-20-prize_2024.md (hunk 2 above)
Comment (odow) [inline _posts/2024-07-20-prize_2024.md:4]:
```suggestion
date:   2024-07-20
categories: [jump-dev-prize]
author: "Benoit Legat and Joaquim Garcia"
```

**Review by odow — APPROVED** [review]

**Comment by odow** [comment]
Do we have photos of this?

**Comment by mlubin** [comment]
I think we should add a sentence citation for each paper (better to discuss privately first).

**Comment by odow** [comment]
Bump @blegat and @joaquimg

**Comment by pulsipher** [comment]
Thanks, for putting this together and for the award! Would you please update my presentation title to match the updated one on the YouTube video?

**Comment by odow** [comment]
Updated

**Comment by blegat** [comment]
> Do we have photos of this?

I added a photo

**Comment by odow** [comment]
Still need a prize citation for why you chose these talks :smile:

**Review by odow — APPROVED** [review]
LGTM

**Comment by mlubin** [comment]
LGTM (previously approved over email)

**Pull request merged**